tools/ci/production-install

```diff
@@ -5,7 +5,7 @@
 set -e
 set -x
 
-ZULIP_PATH=~/zulip
+ZULIP_PATH=/home/circleci/zulip
```

Comment:
I think Focal is resolving it as `/root/zulip` because of the fact that we're running as root, so unless the `HOME` environment variable was passed through `sudo`, that's expected.